src/views/Profile.jsx

```diff
@@ -1,7 +1,29 @@
+import { useSelector } from "react-redux";
+
+import "./Profile.css"
+
 const Profile = () => {
+  const rockets = useSelector((state) => state.rockets);
+  const missions = useSelector(state => state.missions);
+
   return (
-    <div>
-      Profile Component
+    <div className="flex-row justify-around Profile">
+      <div style={{width: "35%"}}>
+        <h2>My Missions</h2>
+        <div className="Profile__left">
+          {
+            missions.map(({mission_name, reserved}) => reserved && (<p>{ mission_name }</p>))
+          }
```

Comment:
- [ ] I would recommend rendering a list of all joined missions using the `filter()` method as it is the best approach and it will give you all the filter results.